components/core/src/package/install.rs

```diff
@@ -111,6 +127,57 @@ impl PackageInstall {
         }
     }
 
+    /// Find an installed package that is at minimum the version of the given ident.
+    fn resolve_package_install_min(ident: &PackageIdent,
+                                   fs_root_path: Option<&Path>)
```

Comment:
We should try and get in the habit of using `AsRef<Path>` when we need `&Path`. There's a number of types in the standard library that implement that and it makes for a more usable UI. It's even a pattern in the Path module itself in the standard library ;)